pkg/registry/core/next/network_service_registry_server.go

```diff
@@ -54,7 +54,9 @@ func NewWrappedNetworkServiceRegistryServer(wrapper NetworkServiceRegistryServer
 	}
 	rv := &nextNetworkServiceRegistryServer{servers: make([]registry.NetworkServiceRegistryServer, 0, len(servers))}
 	for _, s := range servers {
-		rv.servers = append(rv.servers, wrapper(s))
+		if s != nil {
```

Comment:
it seems to me that in this way we ignore the problem of incorrectly passed parameters by SDK developer. The developer should not use nil servers/clients in pkg next.

Reply:
I'd tend to agree here... best to fail fast and dramatically in that case.

Reply:
It makes usage of this method much simple, I could just pass different values, and if some of them are nil they will be skipped. I do not see any problems actually.

Reply:
Let's consider the case when you know that you have a chain of A,B(B=nil),C chain elements, and you trying to figure out why my chain element B did not work?
With this check, you can spend some minutes to figure out that B = nil. But without this fix, it will fast panic :)

Reply:
I suggest to debug and check in this case, it is easy enough to be sure for parameters you pass to method.

Reply:
Ok, I will have some filter method before calling each of this ones in this case.

Reply:
Nope, without it it will fail not here, but in Request, and it will be absolutely not clear why.

Reply:
Usage example for it is
```go
// NewNetworkServiceRegistryServer - wraps the provided registry.NetworkServiceRegistryServer in tracing
func NewNetworkServiceRegistryServer(traced registry.NetworkServiceRegistryServer) registry.NetworkServiceRegistryServer {
	return &traceRegistryServer{traced: traced}
}
```
And it is not failed on construction.

Reply:
Hm.. yes, you are right... I missed that it will throw panic in request... 

Probably then we need to add nil panic in pkg next?

```go
if c == nil {
    panic(...)
}
```

What do you think?

Reply:
panic should work, but I still think if person who use this API should be capable to check arguments to be or not nil.

Reply:
ok, I think we can solve this in separate PR.

Reply:
Removed this kind of checks.